mypy/expandtype.py

```diff
@@ -209,7 +209,11 @@ def visit_erased_type(self, t: ErasedType) -> Type:
         return t
 
     def visit_instance(self, t: Instance) -> Type:
-        args = self.expand_types_with_unpack(list(t.args))
+        if len(t.args) == 0:
+            # TODO: Why do we need to create a copy here?
```

Comment:
@JukkaL It is safer to always create new types in `expand_type()` as the caller may sometimes modify the existing type in-place (at least it was the case in the past).